docs/assets/includes/ai-info.md

```diff
@@ -0,0 +1,5 @@
+!!! note
+    -   This feature is compatible with most programming languages and requires no additional setup.
+    -   Suggestions are generated using the description of the static analysis issue and a few lines of surrounding code.
```

Comment:
Hopefully we'll be able to be more specific here. We should clearly state that the lines of code sent to the OpenAI API are the line where the issue was identified plus "x" lines of code above and below that line.

Reply:
I agree. Worst case, we publish this version and follow up with a task immediately.

Reply:
Yes, totally agree!